third_party/bigframes_vendored/pandas/core/indexes/base.py

```diff
@@ -172,44 +396,138 @@ def isin(self, values):
         passed set of values. The length of the returned boolean array matches
         the length of the index.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> idx = bpd.Index([1,2,3])
+            >>> idx
+            Index([1, 2, 3], dtype='Int64')
+
+        Check whether each index value in a list of values.
+
+            >>> idx.isin([1, 4])
+            array([ True, False, False])
+
+            >>> midx = bpd.MultiIndex.from_arrays([[1,2,3],
+            ...                                   ['red', 'blue', 'green']],
+            ...                                   names=('number', 'color'))
+            >>> midx
+            MultiIndex([(1,   'red'),
+                        (2,  'blue'),
+                        (3, 'green')],
+                       names=['number', 'color'])
+
         Args:
             values (set or list-like):
                 Sought values.
 
         Returns:
-            Series: Series of boolean values.
+            bigframes.pandas.Series:
+                Series of boolean values.
+
+        Raises:
+            TypeError:
+                If object passed to ``isin()`` is not a list-like
         """
         raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
 
     def all(self) -> bool:
         """Return whether all elements are Truthy.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+        True, because nonzero integers are considered True.
+
+            >>> bpd.Index([1, 2, 3]).all()
+            True
```

Comment:
Unfortunately, `pandas` gives a `numpy` object in some versions. It should be interchangeable with a bool, but it has a different repr. Try this (here and in other failing tests):

```suggestion
            >>> bool(bpd.Index([1, 2, 3]).all())
            True
```

Reply:
Thanks for the suggestion, I'll give it a try.